dev/sbt-plugin/src/sbt-test/sbt-plugin/run-all-javadsl/p/conf/application.conf

```diff
@@ -1,11 +0,0 @@
-lagom.play {
-  service-name="p"
```

Comment:
Good catch. Being an sbt-test I assumed this feature would've been test s/where else. Reused this test to restore coverage of this feature.